test/benchmarks/README.md

```
@@ -31,6 +31,22 @@ Benchmarks include implementations of:
 By default these benchmarks report things on the nanosecond scale and are small enough
 to measure very minute differences.
 
+*Alternative zlib library*
+
+To benchmark against an alternative zlib-compatible library, use the `ZLIB_LIBRARY`
+CMake argument. When set, only the public API benchmarks are built:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)